docs/changelogs/v0.27.md

```diff
@@ -28,10 +29,14 @@ You can now configure the upper-bound of a cached IPNS entry's Time-To-Live via
 
 #### RPC client: deprecated DHT API, added Routing API
 
-The RPC client now includes a Routing API to match the available commands in `/api/v0/routing`. In addition, the DHT API has been marked as deprecated. 
+The RPC client now includes a Routing API to match the available commands in `/api/v0/routing`. In addition, the DHT API has been marked as deprecated.
```

Comment:
@hacdias since we remove `/api/v0/dht` in this release, we need to fixup GO client.

Mind doing the same thing we did in https://github.com/ipfs/js-kubo-rpc-client/pull/215 and switch deprecated DHT places like [this](https://github.com/ipfs/kubo/blob/dccbfcf6b5e5ff27fe164f54baeb00622b6a3286/client/rpc/dht.go#L20) to `routing/` in this PR? (so its deprecated, but still works)

```suggestion
The RPC client for GO (`kubo/client/rpc`) now includes a Routing API to match the available commands in `/api/v0/routing`. In addition, the DHT API has been marked as deprecated and use `/api/v0/routing` behind the scenes.
```